build.xml

```diff
@@ -66,20 +85,44 @@
 
     <target name="build"
             description="Compiles all sources to the classes directory">
-	<javac includeantruntime="false" deprecation="yes"
-			source="11" target="11"
-			optimize="true" srcdir="tests"
+	    <javac includeantruntime="false" deprecation="yes"
+			source="11" release="11"
```

Comment:
`release="11"` is there replace `source="11" target="11"` (one attribute is used instead of two)

also resolve merge conflicts, please, so I can merge it

Reply:
done